docker/tomcat-debug.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.8'
+
+services:
+    ff-test:
+        volumes:
+            - "../test/src/test/testtool:/opt/frank/testtool-ext"
+            - "../core/target/ibis-adapterframework-core-$VERSION.jar:/usr/local/tomcat/webapps/iaf-test/WEB-INF/lib/ibis-adapterframework-core-$VERSION.jar:ro"
```

Comment:
Je neemt nu alleen de core jar over, dan kan je beter alle frameworks jar mounten...

Reply:
Zoals besproken, is de core.jar een voorbeeld. De andere jars zitten overal en nergens. Een handige developer kan hiermee zelf verder.